tests/playwright-test/reporter-html.spec.ts

```diff
@@ -2898,6 +2898,24 @@ for (const useIntermediateMergeReport of [true, false] as const) {
       const prompt = await page.evaluate(() => navigator.clipboard.readText());
       expect(prompt, 'contains snapshot').toContain('- button "Click me"');
     });
+
+
+    test('locator.fill value is rendered', async ({ runInlineTest, showReport, page }) => {
```

Comment:
I'd rather amend `should report api steps` from `test-step.spec.ts`. This functionality is not specific to the html report.

Reply:
done in https://github.com/microsoft/playwright/pull/35457/commits/5265392411533b78329a039e83970a68311292b3